plugin.video.hdaustria/resources/settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
would be nice to get this updated to the new style settings once you have a chance to do it

Reply:
Good point, I'll take a look at this later.